Tests/ValetIntegrationTests/SinglePromptSecureEnclaveIntegrationTests.swift

```diff
@@ -75,7 +75,7 @@ class SinglePromptSecureEnclaveIntegrationTests: XCTestCase
         }
         
         try valet().setString(passcode, forKey: key)
-        let equivalentValet = SecureEnclaveValet.valet(with: valet().identifier, accessControl: .devicePasscode)
+        let equivalentValet = SinglePromptSecureEnclaveValet.valet(with: valet().identifier, accessControl: .devicePasscode)
```

Comment:
This makes sense. Though maybe let's rename `equivalentValet` to `similarValet` since they aren't actually equivalent?